pyproject.toml

```
@@ -91,6 +91,7 @@ optional = [
   "scikit-learn>=0.24.2",  # optuna/importance.
   "scipy",  # optuna/_gp.
   "torch",  # optuna/_gp.
+  "greenlet",  # optuna/_gp. necessary for faster gp. Currently greenlet is included as a dependency of SQLAlchemy.
```

Some environments don't include greenlet in their SQLAlchemy dependencies, so how about removing this comment? https://docs.sqlalchemy.org/en/20/faq/installation.html

```suggestion
  "greenlet",  # optuna/_gp. necessary for faster gp.
```

---

Source code comments are written to help future readers and modifers understand the code, so describing this particular change could be misleading. (Such comments should remain on the PR instead.) Here, simply commenting where greenlet is used is sufficient.

---

OK, I understand. I'll remove the comment. Thank you for the feedback!